### ISSUE TITLE
Proposal: Content Dialogs without a description

### ISSUE THREAD

**Issue by MarcAnt01** [microsoft/microsoft-ui-xaml#3180]
# Proposal: [Content Dialogs without a description] 

Content Dialogs without a description

## Summary

I'd like to use Content Dialogs without a description, but currently that results in a blank space which is annoying,

## Rationale

Users should be free to choose if their dialogs require a description or not. For example in a "Do you want to save" dialog, description is useless. I'd rather not have one.

## Scope

| Capability | Priority |
| :---------- | :------- |
| This proposal will allow developers to accomplish Y | Could |

**Comment by yaichenbaum** [comment]
Removing the blank space is something I would like to see as well, this an issue I sometimes run into even with a description in the dialog. An example where I recently had this issue was with a confirmation dialog for deleting files. The dialog still had a description letting the user know the number of files selected, however it was very short and there was unnecessary blank space in the dialog.

**Comment by MarcAnt01** [comment]
Yeah, I think having a minimum eight that cannot be reduced makes no sense at all, too
